src/module.ts

```diff
@@ -30,7 +30,7 @@ export default defineNuxtModule({
     configKey: 'tailwindcss'
   },
   defaults: nuxt => ({
-    configPath: 'tailwind.config.js',
+    configPath: 'tailwind.config',
```

Comment:
this is the main change, jiti will handle `.ts` and `.js` among other extensions presumebly

Reply:
This broke the default config so it no longer loads tailwind.config.js by default